app/routes/publications/publication.js

```diff
@@ -9,8 +9,7 @@ export default class PublicationRoute extends Route {
     return this.store.queryOne('publication-flow', {
       'filter[:id:]': params.publication_id,
       include: [
-        'case',
-        'case.subcases',
+        'case.decisionmaking-flow.subcases',
         'decision-activity',
```

Comment:
If the check in `getIsViaCouncilOfMinisters()` in the publication-service changes, this include might also be overkill.

Reply:
Seems like `decisionAcitvity` is still used in the decision-related route and components. E.g. `DecisionsInfoPanel` uses it a bunch. Might be worth it to keep it in for that.